README.md

```diff
@@ -360,10 +360,11 @@ Epics with some known stories to be flushed out in the semifinalist stage, with
 ### k. Risks and Mitigations
 
 1. Vendor Lock-in: leveraging AWS Amplify and Serverless introduces the risk of vendor lock-in. We believe the benefits outweight the risks, as both services significantly reduce the operation cost, and free the development team from setting up the infrastructure and the DevOps toolkit.
-  - To mitigate the risk of lock-in, we recommend coding the application using the vendor-agnostic blocks, and not using Amplify-specific modules.
+  - To mitigate the risk of lock-in, we recommend identifying which amplify-specific modules are in use, in case a future migration is required
+  - In particular, the cognito backend deployed by amplify will require significant efforts to migrate local users and permissions. This is a problem that is particular to all major identity and authentication systems.
 
 2. Third-Party Library Updates: Amplify and frontend libraries require frequent security updates to mitigate the exploitation of security vulnerabilities. 
-  - To mitigate this risk, we recommend leveraging a DevSecOps pipeline, as well as enabling certain AWS security measures.  
```

Comment:
I still feel having a devsecops pipeline helps with the automatic security scanning and identifying the critical security risks (in particular for frontend libraries), and act upon them. What do you think?

Reply:
We can do all that within the amplify build process, you can run pre and post build hooks, which can include sending code over. We could also hook in and do it on commit in parallel, as we should be using feature branches anyway.

Amplify is providing the pipeline in this case.

Reply:
btw you can even call API's or download what you need in the build process, too.

Reply:
Does it worth mentioning? I don't have a strong opinion, but I feel like saying that we thought about DevSecOps would be great (along the line of the Security AC).

Reply:
Yeah showing that we've thought a devsecops pipeline shows forward thinking. I don't think that we should remove it